<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FBrowserMainWindow</class>
 <widget class="QMainWindow" name="FBrowserMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FBrowser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../icons/browser-internet-svgrepo-com.svg</normaloff>../../icons/browser-internet-svgrepo-com.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777210</width>
        <height>16777210</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Home Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWebView" name="webView">
          <property name="url">
           <url>
            <string>http://www.google.com/#spf=1621609919283</string>
           </url>
          </property>
          <property name="renderHints">
           <set>QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionprevious_page"/>
   </widget>
   <widget class="QMenu" name="menuNaviagtion">
    <property name="title">
     <string>Naviagtion</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNaviagtion"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionprevious_page"/>
   <addaction name="actionnext_page"/>
   <addaction name="actionstop_page_loading"/>
   <addaction name="actionrefresh_current_page"/>
   <addaction name="actionhome_page"/>
  </widget>
  <action name="actionprevious_page">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/arrow-back-outline-svgrepo-com.svg</normaloff>../../icons/arrow-back-outline-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>previous_page</string>
   </property>
  </action>
  <action name="actionnext_page">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/arrow-forward-outline-svgrepo-com.svg</normaloff>../../icons/arrow-forward-outline-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>next_page</string>
   </property>
  </action>
  <action name="actionstop_page_loading">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/minus-circle-outline-svgrepo-com.svg</normaloff>../../icons/minus-circle-outline-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>stop_page_loading</string>
   </property>
  </action>
  <action name="actionrefresh_current_page">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/refresh-outline-svgrepo-com.svg</normaloff>../../icons/refresh-outline-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>refresh_current_page</string>
   </property>
  </action>
  <action name="actionhome_page">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/globe-outline-svgrepo-com.svg</normaloff>../../icons/globe-outline-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>home_page</string>
   </property>
  </action>
  <action name="actiongo_to_address">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/next-svgrepo-com.svg</normaloff>../../icons/next-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>go_to_address</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
